src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentService.kt

```diff
@@ -390,21 +408,23 @@ class AppointmentService(
     return rescheduled
   }
 
-  private fun setAttendanceAndBehaviourIfHistoricAppointment(
+  private fun setAttendanceAndSessionFeedbackIfHistoricAppointment(
     appointment: Appointment,
     attended: Attended?,
-    additionalAttendanceInformation: String?,
-    behaviourDescription: String?,
+    attendanceFailureInformation: String?,
+    sessionSummary: String?,
```

Comment:
Why sessionSummary and sessionResponse can be nullable and in later you are checking sessionSummary!! ?

Reply:
sessionSummary!! and sessionResponse!! are wrapped in an attended?.let {} block - If the user attended then they would continue to the Session Feedback page where both sessionSummary and sessionResponse fields are required. Otherwise, they would submit the form by only needing to complete the Attendance Feedback page.

Reply:
Ok understood, thanks for the clarification